packages/wdio-applitools-service/package.json

```diff
@@ -35,7 +35,8 @@
   },
   "dependencies": {
     "@wdio/eyes.webdriverio": "^2.0.2",
-    "@wdio/logger": "^5.4.6"
+    "@wdio/logger": "^5.4.6",
+    "geckodriver": "1.14.1"
```

Comment:
bootstrap and CI failing are failing because of new geckodriver release, had to hardcode it here. See https://github.com/vladikoff/node-geckodriver/issues/59

Reply:
Why is this dependency needed for the service? The CI issue should be fixed in a separate PR first.

Reply:
Is it a proper way to fix CI? I can submit PR with this fix if it is suitable
@christian-bromann

Reply:
@mgrybyk I don't understand why this is added as dependency for the applitools service and how this fixes CI. We don't have `geckodriver` as dependency anywhere.

Reply:
@christian-bromann I don't understand it as well but something is installing chrome and gecko drivers for wdio-applitools-service.
As far as geckodriver 1.15.0 is broken I've decided to switch back to 1.14.1

Reply:
"@wdio/eyes.webdriverio": "^2.0.2",

Reply:
ok, I have to do PR for https://github.com/applitools/eyes.webdriverio.javascript/blob/master/package.json

Reply:
We need to update it [here](https://github.com/christian-bromann/eyes.webdriverio.javascript) then

Reply:
I had to clone the original package because it doesn't support v5 yet. I should have better documented that.

Reply:
ooops, ok :)

Reply:
@christian-bromann https://github.com/christian-bromann/eyes.webdriverio.javascript/pull/1

Reply:
please remove the dependency, it should trigger a new CI run with the new version I released

Reply:
@christian-bromann thanks, I'll update PR in the evening, have to leave now